### ISSUE TITLE
Blend musics

### ISSUE THREAD

**Issue by Phyks** [Polochon-street/leleleplayer#10]
Hi,

Not sure if it is super easy or not to do, but as you can find similar musics, why not trying to make a smooth blending between the musics, to have a continuous play, without having any gap between the musics?

**Comment by sylvainar** [comment]
Yep good idea. Or at list remove the blanks at the end and at the beginning of each music ?
